docs/services/vision/obstacles_distance.md

```diff
@@ -16,7 +16,7 @@ _Changed in [RDK v0.2.36 and API v0.1.118](/appendix/changelog/#vision-service)_
 `obstacles_distance` is a segmenter that takes point clouds from a camera input and returns the average single closest point to the camera as a perceived obstacle.
 It is best for transient obstacle avoidance.
 
-For example, if you have an ultrasonic distance sensor as an [`ultrasonic` camera](/components/camera/ultrasonic/), this model will query the sensor `"num_queries"` times, and then take the average point from those measurements and return that as an obstacle.
+For example, if you have an ultrasonic distance sensor as an [`viam:ultrasonic:camera`](https://github.com/viam-modules/viam-ultrasonic), this model will query the sensor `"num_queries"` times, and then take the average point from those measurements and return that as an obstacle.
```

Comment:
@npentrel, I'm not sure what the correct thing to do here or in the tutorials is.